app/scripts/components/alerts.component.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .component('alerts', {
+    bindings: {
+      // Map (or array) of alert objects, each with properties
+      //   type:     'error', 'warning', 'success', or 'info' (default)
+      //   message:  the message to display
+      //   links:    an array of link objects (optional, see below)
+      //   onClose:  callback function when the user dismisses the alert (optional)
+      //
+      // Each link can have the following properties:
+      //   href:     link URL (optional if onClick is set)
+      //   label:    the link text
+      //   onClick:  handler to call when the link if clicked (optional)
+      //             If onClick returns true, dismisses the alert
+      alerts: '=',
```

Comment:
I think we should switch these to `<` bindings unless there is a reason not to.

(Not sure if this causes a problem with how we handle close since we set the hidden property inside the component. We might need to leave `alerts` as a `=` binding.)